docs/Options.md

```diff
@@ -178,6 +179,24 @@ Git status indicators is shown only when you have dirty repository.
 | `SPACESHIP_GIT_STATUS_BEHIND` | `⇣` | Indicator for unpulled changes (behind of remote branch) |
 | `SPACESHIP_GIT_STATUS_DIVERGED` | `⇕` | Indicator for diverged chages (diverged with remote branch) |
 
+#### Git commit time (`git_commit_time`)
+
+The value is displayed in three possible colors :
+-   green (for short time)
+-   yellow (for medium time)
+-   red (for long time)
+
+Value is considered as "short" if she is smaller than `SPACESHIP_GIT_COMMIT_TIME_THRESHOLD_LOW`, "long"
+if she is greater than `SPACESHIP_GIT_COMMIT_TIME_THRESHOLD_HIGH` or "medium" if she is between them.
+
+| Variable | Default | Meaning |
+| :------- | :-----: | ------- |
+| `SPACESHIP_GIT_COMMIT_TIME_SHOW` | `true` | Show Git commit time subsection |
```

Comment:
I think this section should be **disabled** by default. This section has lot of processing and prefer against incur all this time to prompt by default.